chia/plotting/manager.py

```diff
@@ -407,7 +407,7 @@ def reduce_function(x: Dict, y: Dict) -> Dict:
 
             # First drop all plots we have in plot_filename_paths but not longer in the filesystem or set in config
             def plot_removed(test_path: Path):
-                return not test_path.exists() or test_path.parent not in plot_filenames
+                return not test_path.exists() or test_path.parent not in plot_directories
```

Comment:
this took a while to thread through the change as well, but `plot_directories` ends up being a `Set` of the same values as the previous keys in `plot_filenames`.

Reply:
Correct